Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/GetPageMainPage/GetPageOfStreetcodesMainPageHandler.cs

```diff
@@ -43,9 +48,19 @@ public async Task<Result<IEnumerable<StreetcodeMainPageDTO>>> Handle(GetPageOfSt
                     streetcode.Images = streetcode.Images.Where(x => x.ImageDetails != null && x.ImageDetails.Alt.Equals(keyNumOfImageToDisplay.ToString())).ToList();
                 }
 
-                var streetcodesPaginated = streetcodes.Paginate(request.page, request.pageSize);
+                var shuffledStreetcodes = new List<StreetcodeContent>();
```

Comment:
Do not copy colection of streetcodes, make the previous one as list and sort it in random order. This will make the use of memory optimal.

Reply:
> Do not copy colection of streetcodes, make the previous one as list and sort it in random order. This will make the use of memory optimal.

I removed that and also tried to compare OrderBy and Sort methods.
For some reason, inplace Sort .... feels less random??? Also I didn't see any changes in memory usage OrderBy vs Sort. Could you explain that?
OrderBy allocations:
![1](https://github.com/ita-social-projects/StreetCode/assets/63905892/2a64eae3-865e-4974-8887-b75212bc6fd8)
Sort allocations:
![2](https://github.com/ita-social-projects/StreetCode/assets/63905892/ffb90ab0-4002-461c-b212-66d5e7564321)